guidelines/sc/20/audio-control.html

```diff
@@ -4,8 +4,8 @@ <h4>Audio Control</h4>
    
    <p class="conformance-level">A</p>
    
-   <p>If any audio on a Web page plays automatically for more than 3 seconds, either a <a>mechanism</a> is available to pause or stop the audio, or a mechanism is available to control audio
-      volume independently from the overall system volume level.
+   <p>If any audio on a Web page plays automatically for more than 3 seconds, either a <a>mechanism</a> is available to pause or stop the audio or a mechanism is available to control audio
```

Comment:
personally, think it reads a bit awkard and long without that comma after "stop the audio", but i agree with that added comma in the next line

Reply:
Yea, it has a lot of "or"s, but I think that's the way to make clear it applies to both.